flask_security/core.py

```diff
@@ -88,13 +89,14 @@
     'CHANGE_SALT': 'change-salt',
     'REMEMBER_SALT': 'remember-salt',
     'DEFAULT_REMEMBER_ME': False,
-    'DEFAULT_HTTP_AUTH_REALM': 'Login Required',
-    'EMAIL_SUBJECT_REGISTER': 'Welcome',
-    'EMAIL_SUBJECT_CONFIRM': 'Please confirm your email',
-    'EMAIL_SUBJECT_PASSWORDLESS': 'Login instructions',
-    'EMAIL_SUBJECT_PASSWORD_NOTICE': 'Your password has been reset',
-    'EMAIL_SUBJECT_PASSWORD_CHANGE_NOTICE': 'Your password has been changed',
-    'EMAIL_SUBJECT_PASSWORD_RESET': 'Password reset instructions',
+    'DEFAULT_HTTP_AUTH_REALM': _('Login Required'),
```

Comment:
Thanks it does, the login_manager from Flask-Login has a gettext callback that can be used, and I just have to find something similar and appropriate for the forms, and make sure its all done properly everywhere.

My test bed is up and running, so I can figure things out. A bit busy this week, but I'll have something by the weekend.

Thanks for helping out a guy with his first open source pull request. Kinda exciting. :)